sound/soc/sof/ipc3-topology.c

```diff
@@ -1727,6 +1728,199 @@ static int sof_ipc3_control_free(struct snd_sof_dev *sdev, struct snd_sof_contro
 	return sof_ipc_tx_message(sdev->ipc, fcomp.hdr.cmd, &fcomp, sizeof(fcomp), NULL, 0);
 }
 
+/* send pcm params ipc */
+static int ipc_pcm_params(struct snd_sof_widget *swidget, int dir)
```

Comment:
Would it make sense to rename this and the `ipc_trigger()` to be less generic looking, like `sof_ipc3_keyword_pcm_params()` / `_trigger()`
We have similar function names for pcm hw_params and trigger...